packages/Webkul/Admin/src/Resources/lang/en/app.php

```diff
@@ -1553,7 +1553,8 @@
             'generate-invoice'                      => 'Automatically generate the invoice after placing an order',
             'set-invoice-status'                    => 'Set the invoice status after creating the invoice to',
             'set-order-status'                      => 'Set the order status after creating the invoice to',
-            'generate-invoice-applicable'           => 'Applicable if automatic generate invoice is enabled'
+            'generate-invoice-applicable'           => 'Applicable if automatic generate invoice is enabled',
+            'records-found'                         => 'Record(s) found'
```

Comment:
Please provide all translations present in the lang folder.

Reply:
translations pushed for all languages available